src/main/java/com/functions/DistributedTracingFunction.java

```diff
@@ -0,0 +1,77 @@
+package com.function;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.HttpMethod;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.CosmosDBOutput;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Optional;
+import java.util.Random;
+
+public class DistributedTracingFunction {
```

Comment:
Samples looks good. Minor suggestion, it will be good to have some comments on what customers can expect to see in portal when using this sample.  
Similar to [here](https://github.com/Azure-Samples/azure-functions-samples-java/blob/master/src/main/java/com/functions/BlobTriggerFunction.java#L13)